enhancements/machine-api/machine-deletion-hooks.md

```diff
@@ -0,0 +1,412 @@
+---
+title: machine-deletion-hooks
+authors:
+  - "@JoelSpeed"
+  - "@michaelgugino"
+reviewers:
+  - "@enxebre"
+  - "@elmiko"
+  - "@ademicev"
+  - "@hexfusion"
+approvers:
+  - TBD
```

Comment:
I would expect someone from the cluster infrastructure team to be listed here. If you need a neutral moderator, let me know and I can help find someone.

Reply:
A neutral approver would be handy. I have already mentioned this to @deads2k as I was asking him some questions about finalizers, his expertise may make him an appropriate approver here